CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 # Set C and C++ standard version
 set(CMAKE_C_STANDARD 99)
 set(CMAKE_C_STANDARD_REQUIRED True)
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
FYI I noted this in the changelog. Better now to include the changelog update in this PR as it will cause endless conflicts. Please do rebase on the latest develop though for ease of review.

Reply:
You mean this should be noted in the CHANGELOG at the very end after having finished this PR, and not now? I'll include it in the updated to-do list.

This PR is already rebased on the latest `develop`, so I'm not sure what you are referring to?

Reply:
I mean that you don't need to worry about it because I've already done it.

Reply:
Ah, right, like that, perfect, thank you!